File: windows/WidgetProvider.Core/Quote.cs
namespace ColdOpen.WidgetProvider.Core;

/// <summary>
/// A single cold-open narration line. Mirrors the schema in
/// <c>corpus/schema.json</c> at the repo root — keep the two in sync if
/// either changes.
/// </summary>
public sealed record Quote(string Text, string EpisodeTitle, int Season, int Episode, string Arc);

File: android/core/src/main/kotlin/com/coldopen/core/Quote.kt
package com.coldopen.core

/**
 * A single cold-open narration line. Mirrors the schema in `corpus/schema.json`
 * at the repo root — keep the two in sync if either changes.
 */
data class Quote(
    val text: String,
    val episodeTitle: String,
    val season: Int,
    val episode: Int,
    val arc: String,
)
